perl-lib/OESS/lib/OESS/Cloud.pm

```diff
@@ -145,6 +146,18 @@ sub setup_endpoints {
 
             my $azure = OESS::Cloud::Azure->new();
 
+            # Configure peering information only on layer 3
+            # connections.
+            my $peering;
+            if (defined $ep->vrf_endpoint_id) {
+                $peering = {
+                    vlan             => $ep->tag,
+                    local_asn        => 55038,
```

Comment:
shouldn't the local_asn come from the configuration... I'm pretty sure we specify it ther